jest.setup.js

```diff
@@ -2,3 +2,13 @@ import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
```

Comment:
Can this be removed now that we have switched to `react-testing-library`?

Reply:
Yes the whole jest setup can be removed as well as the tests that are testing if the `propTypes.required` works in which it should already be expected.